docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -66,13 +80,42 @@ kubectl apply -f my-medusa-bucket-key.yaml
 secret/medusa-bucket-key configured
 ```
 
-### Create or update the k8ssandra
+### Create or update the k8ssandra cluster
+
+Install the `k8ssandra` chart with the following properties. You can reference the provided [backup-restore-values.yaml](./backup-restore-values.yaml) file. It contains:
+
+```
+size: 3
+backupRestore: 
+  medusa:
+    enabled: true
+    bucketName: k8ssandra-bucket-dev
+    bucketSecret: medusa-bucket-key
+    multiTenant: true
+    storage: s3
+```
+
+The chart's entries relate to a Kubernetes Secret, which contains the object store credentials. 
 
-Install the k8ssandra chart with the following properties. Backup and restore operations are enabled by default. In the following example, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
+Example:
 
-`helm install k8ssandra-1 k8ssandra/k8ssandra --set backupRestore.medusa.bucketName=k8ssanda-bucket-dev --set backupRestore.medusa.bucketSecret=medusa-bucket-secret`
+`helm install k8ssandra k8ssandra/k8ssandra -f backup-restore-values.yaml`
 
-The `k8ssandra` Helm chart includes cluster services and the Grafana Operator. Notice that `k8ssandra` add a number of properties in the `cassdc` datacenter.  
+Allow a few minutes for the pods to start and proceed to a Ready state; check the pod status periodically:
+
+```
+kubectl get pods                              
+NAME                                                         READY   STATUS    RESTARTS   AGE
+cass-operator-86d4dc45cd-8p7cq                               1/1     Running   0          98s
+k8ssandra-tools-kube-prome-operator-6bcdf668d4-b2r6v         1/1     Running   0          98s
+.
+.
+.
+```
+
+Backup and restore operations are enabled by default. In the example YAML, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
+
+The `k8ssandra` Helm chart includes the Grafana Operator. Notice that `k8ssandra` adds a number of properties in the `cassdc` datacenter.  
 
 `kubectl get cassdc dc1 -o yaml`
```

Comment:
Checking for `cassandrOperatorProgress: Ready` is not always sufficient. The CassandraDatacenter status reports several conditions. The user should also look for the following condition in the status:

```yaml
status:
  cassandraOperatorProgress: Ready
  conditions:
  ...
  - lastTransitionTime: "2021-02-03T17:04:52Z"
    message: ""
    reason: ""
    status: "True"
    type: Ready
  ...
```

Reply:
Incorporated - thanks